erpnext_shipping/erpnext_shipping/shipping.py

```diff
@@ -50,7 +50,7 @@ def fetch_shipping_rates(
 		if delivery_to_type != "Company":
 			delivery_contact = get_contact(delivery_contact_name)
 		else:
-			delivery_contact = get_company_contact(user=pickup_contact_name)
+			delivery_contact = get_contact(delivery_contact_name)
 			delivery_contact.email_id = delivery_contact.pop("email", None)
```

Comment:
```suggestion
			delivery_contact = get_contact(delivery_contact_name)
```

I think we also need to remove this line, which was used to rename the `"email"` property from **User** to `"email_id"`, in order to make the data structure similar to **Contact**. Since we are now fetching a **Contact** directly, this is no longer necessary.